### PR TITLE
Add option to specify branch

### PR DESCRIPTION
**Pull request by petermorlion**
We want to use the email notification feature only on our master branch. This PR should make this possible. Essentially, it just checks for the existence of the option, and if it is there, it checks if we're currently on that branch. If not, we bail out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Send an email message (note: requires python 2.x)
 * `passed-body` - Use this option to specify the passed body.
 * `failed-body` -  Use this option to specify the failed body.
 * `on` - Possible values: `always` and `failed`, default `always`
+* `branch` - Use this option to specify the branch for which an email should be sent.
+             If not specified, the email will be sent for builds of all brances.
```
Comment (kokaz) [inline README.md]: `s/brances/branches/`

File: run.sh [hunk 2]
```diff
@@ -76,4 +76,11 @@ if [ "$WERCKER_EMAIL_NOTIFY_ON" = "failed" ]; then
   fi
 fi
 
+if [ "$WERCKER_EMAIL_NOTIFY_BRANCH" ]; then
```
Comment (kokaz) [inline run.sh]: Can you use `-n` here like that:
```bash
  if [ -n "$WERCKER_EMAIL_NOTIFY_BRANCH" ]; then 
```

**Comment by petermorlion** [comment]
No problem! I've force pushed the changes.

**Comment by kokaz** [comment]
Thanks for your contribution @petermorlion!

**Pull request merged**